snuba/migrations/migration.py

```diff
@@ -157,9 +157,11 @@ def forwards(self, context: Context, dry_run: bool = False) -> None:
 
         for op in ops:
             if op in local_ops:
-                op.execute(local=True)
+                op.target = OperationTarget.LOCAL
```

Comment:
This is temporary right now right? until we add this to the migrations themselves?

Reply:
yes